openupgrade_scripts/scripts/project/16.0.1.2/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,101 @@
+---Models in module 'project'---
+obsolete model project.delete.wizard [transient]
+# NOTHING TO DO: wizard removed
+
+---Fields in module 'project'---
+project      / account.analytic.tag     / task_ids (many2many)          : DEL relation: project.task
+project      / project.task             / analytic_tag_ids (many2many)  : DEL relation: account.analytic.tag
+# NOTHING TO DO: lost feature
+
+project      / project.milestone        / task_ids (one2many)           : NEW relation: project.task
+project      / project.project          / task_properties_definition (properties_definition): NEW
+project      / project.task             / task_properties (properties)  : NEW hasdefault: compute
+project      / res.company              / analytic_plan_id (many2one)   : NEW relation: account.analytic.plan, hasdefault: compute
+# NOTHING TO DO: new features
+
+project      / project.tags             / project_ids (many2many)       : NEW relation: project.project
+project      / project.tags             / task_ids (many2many)          : NEW relation: project.task
+# NOTHING TO DO: reverse relation that already exists on project.project and project.task
+
+pad_project  / project.project          / description_pad (char)        : DEL
+pad_project  / project.project          / use_pads (boolean)            : DEL
+pad_project  / project.task             / description_pad (char)        : DEL
+# TODO: Module pad_project has been removed
```

Comment:
@pedrobaeza I'm not sure what to do with this. Should I rename the table with a legacy name to ensure that data are not deleted ? Or should I just do nothing ?

Reply:
I would say "Nothing to do", as the content is dumped on save to the description fields, and the collaboration comes now from web editor itself.